### DISCUSSION TITLE
kpanda/user-guide/configmaps-secrets/use-configmap/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2980] [Announcements]
# kpanda/user-guide/configmaps-secrets/use-configmap/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/configmaps-secrets/use-configmap/

**Comment by LLLzxx** [comment]
"用作容器数据卷" 下方的第一句描述有误“使用配置项作为容器的环境变量”

**Reply by lemontrees123** [reply]
已更正
